packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -552,7 +555,9 @@ export class StripeProductsAndPlansConverter {
                   planConfig,
                   productConfigId as string,
                   existingPlanConfigId,
-                  getFilePath(plan.id)
+                  getFilePath(
+                    `plan_${plan.currency}_${plan.interval}_${plan.interval_count}_${plan.amount}_${plan.nickname}`
```

Comment:
Having `nickname` at the back seems to cause some issues. I'm getting issues like this one.

```
ERROR fxa-auth-server.StripeProductsAndPlansConverter.convertPlanError: {"error":"ENOENT: no such file or directory, open 'payments-products-config-json/paypal_funding source error/plan_usd_day_1_10203_this_amount corresponds to the paypal error code 10203. see https:/developer.paypal.com/docs/api-basics/sandbox/nt-classic/#test-amount-related-errors.json'","stripePlanId":"price_1IU1oHBVqmGyQTMa0L16D1DR","stripeProductId":"prod_J6EGmJVK3Aaqdr"}
```

Reply:
Looks like amount can also be null, I've subbed in defaults for them in that case.